FusionIIIT/applications/hostel_management/models.py

```diff
@@ -115,13 +127,13 @@ class GuestRoomBooking(models.Model):
     'nationality' stores the nationality of the guests.
     """    
     hall = models.ForeignKey(Hall, on_delete=models.CASCADE)
-    intender = models.ForeignKey(User, on_delete=models.CASCADE)
+    intender = models.ForeignKey(ExtraInfo, on_delete=models.CASCADE)
     guest_name = models.CharField(max_length=100)
     guest_phone = models.CharField(max_length=15)
     guest_email = models.CharField(max_length=40, blank=True)
     guest_address = models.TextField(blank=True)
-    rooms_required =  models.IntegerField(default=1,null=True,blank=True)
-    guest_room_id = models.ManyToManyField(GuestRoomDetail)
+    rooms_required =  models.IntegerField(default=1, null=True, blank=True)
+    guest_room_id = models.ManyToManyField(GuestRoomDetail, blank=True)
```

Comment:
What is the need to define a recursive relationship?

Reply:
This will be removed.